samples/apps/copilot-chat-app/webapp/src/components/chat/ChatInput.tsx

```diff
@@ -63,7 +63,7 @@ interface ChatInputProps {
 export const ChatInput: React.FC<ChatInputProps> = (props) => {
     const { isTyping, onSubmit } = props;
     const classes = useClasses();
-    const { instance, accounts } = useMsal();
+    const { instance, accounts, inProgress } = useMsal();
     const account = useAccount(accounts[0] || {});
```

Comment:
```suggestion
    const account = instance.useActiveAccount();
```
nit

Reply:
@teresaqhoang , @gitri-ms - let's file separate bug for this suggestion.

Reply:
I agree, but it looks like there are several places we are doing this. Let me do it as a separate PR so I can make sure I don't miss any without holding this one up.